lib/reporter/magellan-i18n-reporter.js

```diff
@@ -0,0 +1,206 @@
+const BaseReporter = require( 'testarmada-magellan' ).Reporter;
```

Comment:
Rather than maintain a separate reporter for the i18n screenshots can we just set an envvar like `SCREENSHOTDIR` and have it default to `/screenshots` and just override it here to `screenshots-i18n`?